### PR TITLE
WRO-12412: Fixed ImageItem to have proper size with importing VirtualList

### PR DESCRIPTION
**Pull request by MikyungKim**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
ImageItem would have the wrong size of an image when it's horizontal with importing VirtualList or Dropdown like below.
image

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
The root cause of this issue is that VirtualList imports ImageItem.module.less in useThemeVirtualList.js to apply the scale class to ImageItem when `snapToCenter` is true.
When webpack bundles css, the sandstone/ImageItem/ImageItem.module.less is imported first for the above reason and then ui/Imageitem/ImageItem.module.less imported so the issue occurred.
To resolve this, I made `useThemeVirtualList.module.less` to have the scale class instead of importing ImageItem.module.less.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
With this change, the scale effect is going to apply to any ImageItem instead of "vertical, full image" only.

### Links
[//]: # (Related issues, references)
WRO-12412

### Comments
Enact-DCO-1.0-Signed-off-by: Mikyung Kim (mikyung27.kim@lge.com)

### CODE REVIEW

File: VirtualList/useThemeVirtualList.module.less [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+.scaled {
+	transform: @sand-imageitem-vertical-focus-image-transform;
+}
+
+.applySkins({
+	.scaled {
+		box-shadow: @sand-imageitem-focus-bg-shadow;
+	}
+});
```
Comment (jeonghee27) [inline VirtualList/useThemeVirtualList.module.less:14]: Why don't we remove this styles on ImageItem.module.less?

File: VirtualList/useThemeVirtualList.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ import {useEventKey, useEventFocus} from './useEvent';
 import usePreventScroll from './usePreventScroll';
```
Comment (jeonghee27) [inline VirtualList/useThemeVirtualList.js:13]: I think we need change log

Comment (MikyungKim) [reply VirtualList/useThemeVirtualList.js:13]: I agree. I'll put it right before merge.

File: VirtualList/useThemeVirtualList.module.less (hunk 1 above)
Comment (MikyungKim) [reply VirtualList/useThemeVirtualList.module.less:14]: 👍

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**